### PR TITLE
i18n(zh-cn): Fix typos and add UI strings

### PR DESCRIPTION
**Pull request by zihan-ch**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- Translated content
- Changes to the docs site code

#### Description
Add some UI Strings, create `leftSidebar.ponsoredBy` UIString, fix some minor contents.

### CODE REVIEW

**Comment by zihan-ch** [comment]
I updated the `base` config problem that #1643 has showed. It is not about the zh-cn i18n, but I still resolved it in this PR.

**Comment by zihan-ch** [comment]
Also added the corresponding translation of the config `base` update.

**Comment by Jutanium** [comment]
Hey, thanks for tackling #1643 - please open a separate PR for those `base` configuration docs changes, and leave that translation for later. This will allow us to edit and finalize the English version independent of the translations in this PR!

**Review by delucis — COMMENTED** [review]
Thanks for this @zihan-ch! I’ve left a few suggestions.

File: src/i18n/zh-cn/ui.ts [hunk 1]
```diff
@@ -31,25 +32,40 @@ export default UIDictionary({
 	// `<ThemeToggleButton>` acessibility labels
 	'themeToggle.useLight': '使用浅色主题',
 	'themeToggle.useDark': '使用深色主题',
-	'rightSidebar.github': 'GitHub 上的 Astro 文档',
 	// Used in previous/next page links at the bottom of pages
 	'articleNav.nextPage': '下一页',
-	'articleNav.prevPage': '返回',
+	'articleNav.prevPage': '上一页',
```
[the remark this answers is not in the document]
Comment (zihan-ch) [reply src/i18n/zh-cn/ui.ts:37]: `Back` is fine, but I think it seems more reasonable to use `Previous Page`. The UIString key `'articleNav.prevPage'` is telling us it must be `Previous Page`, isn't it? I feel a little confused.

[the remark this answers is not in the document]
Comment (delucis) [reply src/i18n/zh-cn/ui.ts:37]: You can think of the key on the left as saying “Where is this UI string used?” so sometimes the key won’t 100% match the actual text label. So in this case the label is used on the previous page article navigation arrow, but we chose to use “Back” for the label.

File: src/pages/zh-cn/guides/server-side-rendering.md [hunk 2]
```diff
@@ -105,13 +101,13 @@ if(!product) {
...
 <html>
-  <!-- Page here... -->
+  <!-- 你的页面 -->
 </html>
 ```
 
 #### API 路由
 
-`src/pages/` 文件夹中的 `.js` 或 `.ts` 文件都是 API 路由，它接收 [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) 并返回 [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response)数据。
+`src/pages/` ，目录中的 `.js` 或 `.ts` 文件都是 API 路由，它接收 [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) 并返回 [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response)数据。
```
Comment (delucis) [inline src/pages/zh-cn/guides/server-side-rendering.md:110]: Making a quick fix here before merging!

```suggestion
`src/pages/`，目录中的 `.js` 或 `.ts` 文件都是 API 路由，它接收 [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) 并返回 [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response)数据。
```

**Pull request merged**